config/routes.rb

```diff
@@ -1,87 +1,123 @@
-#                                                                       POST   /api/v2/sessions(.:format)                                                                                                       api/v2/sessions#create
-#                                                    api_v2_site_status PATCH  /api/v2/site_statuses/:id(.:format)                                                                                              api/v2/site_statuses#update
-#                                                                       PUT    /api/v2/site_statuses/:id(.:format)                                                                                              api/v2/site_statuses#update
-#                                         approve_api_v2_access_request POST   /api/v2/access_requests/:id/approve(.:format)                                                                                    api/v2/access_requests#approve
-#                                                api_v2_access_requests GET    /api/v2/access_requests(.:format)                                                                                                api/v2/access_requests#index
-#                                                                       POST   /api/v2/access_requests(.:format)                                                                                                api/v2/access_requests#create
-#                                                 api_v2_access_request GET    /api/v2/access_requests/:id(.:format)                                                                                            api/v2/access_requests#show
-#                                                                       PATCH  /api/v2/access_requests/:id(.:format)                                                                                            api/v2/access_requests#update
-#                                                                       PUT    /api/v2/access_requests/:id(.:format)                                                                                            api/v2/access_requests#update
-#                                                             error_500 GET    /error_500(.:format)                                                                                                             error#error_500
-#                                                            error_nodb GET    /error_nodb(.:format)                                                                                                            error#error_nodb
+# logstash not configured, falling back to standard Rails logging
+# Raven 2.12.2 configured not to capture errors: DSN not set
+# DEPRECATION WARNING: Using `bin/rake routes` is deprecated and will be removed in Rails 6.1. Use `bin/rails routes` instead.
+#  (called from load at /Users/Dan/.rbenv/versions/2.6.1/bin/rake:23)
```

Comment:
:roll_eyes:

Reply:
This was autogenerated so ive manually removed.